scripts/freeze_graph.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+
+import tf_slim as slim
+import tensorflow as tf
+print(tf.__version__)
+
+from tensorflow.python.tools import optimize_for_inference_lib
+from deepvariant.modeling import get_model
+
+parser = argparse.ArgumentParser(description='Script to create a frozen graph for DeepVariant models')
+parser.add_argument('--checkpoint', required=True, help='Path to model.ckpt')
+parser.add_argument('--output', required=True, help='Path to output .pb file')
+parser.add_argument('--moving_average_decay', default=0.9999, help='The decay to use for the moving average')
+args = parser.parse_args()
+
+model = get_model('inception_v3')
+
+out_node = 'InceptionV3/Predictions/Reshape_1'
+in_node = 'input'
+
+# https://github.com/google/deepvariant/blob/r1.0/deepvariant/dv_constants.py
+inp = tf.compat.v1.placeholder(shape=[1, 100, 221, 6], dtype=tf.float32, name=in_node)
```

Comment:
Note that not all models are the same shape. For example, our PacBio model has 8 channels. I recommend making this into an argument that can be changed.

Reply:
Indeed! Thanks